cmd/lakectl/cmd/root.go

```diff
@@ -679,5 +681,6 @@ func initConfig() {
 	viper.SetDefault("local.skip_non_regular_files", false)
 	viper.SetDefault("credentials.provider.aws_iam.token_ttl_seconds", defaultTokenTTL)
 	viper.SetDefault("credentials.provider.aws_iam.url_presign_ttl_seconds", defaultURLPresignTTL)
+	viper.SetDefault("credentials.provider.aws_iam.refresh_interval", defaultRefreshInterval)
```

Comment:
No reason to define an explicit default here, the default of this type is already 0.
The struct instance will contain the default of `time.Duration` if it's not defined here, and it's `0`.